### PR TITLE
Update Functions public API docs for FCM Token instead of IID

### PR DESCRIPTION
**Pull request by paulb777**
Similar changes are needed for the Codable APIs introduced in #8854

#no-changelog

### CODE REVIEW

File: FirebaseFunctions/Sources/Public/FirebaseFunctions/FIRHTTPSCallable.h [hunk 1]
```diff
@@ -47,12 +47,12 @@ NS_SWIFT_NAME(HTTPSCallable)
...
  * The request to the Cloud Functions backend made by this method automatically includes a
- * Firebase Instance ID token to identify the app instance. If a user is logged in with Firebase
+ * Firebase Installations token to identify the app instance. If a user is logged in with Firebase
  * Auth, an auth ID token for the user is also automatically included.
  *
- * Firebase Instance ID sends data to the Firebase backend periodically to collect information
- * regarding the app instance. To stop this, see `[FIRInstanceID deleteIDWithHandler:]`. It
- * resumes with a new Instance ID the next time you call this method.
+ * Firebase Installations sends data to the Firebase backend periodically to collect information
+ * regarding the app instance. To stop this, see `Installations.delete()`. It
```
Comment (maksymmalyhin) [inline FirebaseFunctions/Sources/Public/FirebaseFunctions/FIRHTTPSCallable.h:54]: Hm, actually Functions use FCM token now. FCM token will also be refreshed when `Installations.delete()` is called but maybe we should be more precise here. @chliangGoogle WDYT?

**Review by maksymmalyhin — APPROVED** [review]
LGTM, it looks more precise and useful than before, but it is still slightly inaccurate. I guess  @chliangGoogle can help with better wording.

File: FirebaseFunctions/Sources/Public/FirebaseFunctions/FIRHTTPSCallable.h [hunk 2]
```diff
@@ -47,12 +47,12 @@ NS_SWIFT_NAME(HTTPSCallable)
  * Executes this Callable HTTPS trigger asynchronously without any parameters.
  *
  * The request to the Cloud Functions backend made by this method automatically includes a
- * Firebase Instance ID token to identify the app instance. If a user is logged in with Firebase
+ * Firebase Installations token to identify the app instance. If a user is logged in with Firebase
```
Comment (chliangGoogle) [inline FirebaseFunctions/Sources/Public/FirebaseFunctions/FIRHTTPSCallable.h:50]: I think it's called FCM Token now: https://github.com/firebase/firebase-ios-sdk/blob/master/FirebaseMessaging/Sources/Public/FirebaseMessaging/FIRMessaging.h#L261

And it's not part of Firebase installations

File: FirebaseFunctions/Sources/Public/FirebaseFunctions/FIRHTTPSCallable.h (hunk 1 above)
Comment (chliangGoogle) [reply FirebaseFunctions/Sources/Public/FirebaseFunctions/FIRHTTPSCallable.h:54]: Yeah and if function uses token, then they also need to call Messaging.deleteData() to remove FCM token info.
https://github.com/firebase/firebase-ios-sdk/blob/master/FirebaseMessaging/Sources/Public/FirebaseMessaging/FIRMessaging.h#L397

**Review by chliangGoogle — CHANGES_REQUESTED** [review]

**Review by chliangGoogle — APPROVED** [review]

**Pull request merged**